gcb_web_auth/test_migrations.py

```diff
@@ -60,6 +60,7 @@ class DDSEndpointMigrationTestCase(TestMigrations):
     django_application = 'gcb_web_auth'
 
     def setUpBeforeMigration(self, apps):
+        self.apps = apps
```

Comment:
The above line looks like you are trying to save `apps` for later. This particular apps is from before the migration. `self.apps` gets overwritten in `setUp` after `setUpBeforeMigration ()` to be the new apps so this line has no effect. The migration test framework is a little confusing.